skyvern/forge/sdk/workflow/models/block.py

```diff
@@ -1961,57 +2011,87 @@ async def execute(
 
         download_files_path = str(get_path_for_workflow_download_directory(workflow_run_id).absolute())
 
-        s3_uris = []
+        uploaded_uris = []
         try:
             workflow_run_context = self.get_workflow_run_context(workflow_run_id)
-            actual_aws_access_key_id = (
-                workflow_run_context.get_original_secret_value_or_none(self.aws_access_key_id) or self.aws_access_key_id
-            )
-            actual_aws_secret_access_key = (
-                workflow_run_context.get_original_secret_value_or_none(self.aws_secret_access_key)
-                or self.aws_secret_access_key
-            )
-            client = AsyncAWSClient(
-                aws_access_key_id=actual_aws_access_key_id,
-                aws_secret_access_key=actual_aws_secret_access_key,
-                region_name=self.region_name,
-            )
-            # is the file path a file or a directory?
+            files_to_upload = []
             if os.path.isdir(download_files_path):
-                # get all files in the directory, if there are more than 25 files, we will not upload them
                 files = os.listdir(download_files_path)
-                if len(files) > MAX_UPLOAD_FILE_COUNT:
-                    raise ValueError("Too many files in the directory, not uploading")
+                max_file_count = (
```

Comment:
nit: there's no need to do this if else. we can just assume the MAX_UPLOAD_FILE_COUNT is the same